src/es/components/molecules/carouselTwo/CarouselTwo.js

```diff
@@ -372,6 +375,14 @@ export default class CarouselTwo extends Mutation() {
         transform: var(--nav-transform-hover, scale(1.6));
         z-index: 3;
       }
+      :host > section + div > #index {
+        align-self: center;
+      }
+      :host > section + div {
```

Comment:
does this have no side effects at the other examples?
https://mits-gossau.github.io/web-components-toolbox/#CarouselTwo